.travis.yml

```diff
@@ -24,28 +24,26 @@ matrix:
     - php: 5.4
       env: SYMFONY_VERSION=2.8.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.* CS_FIXER=run
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=strict CS_FIXER=run
```

Comment:
The `SYMFONY_DEPRECATIONS_HELPER=weak` line or the strict setting for this build?  IMO you should have at least one build using the strict setting on 2.8 as that will force the tests to fail if any deprecated methods are in use.